tests/conftest.py

```diff
@@ -206,6 +207,10 @@ def rand_one_dut_hostname(request):
         dut_hostnames = random.sample(dut_hostnames, 1)
     return dut_hostnames[0]
 
+@pytest.fixture(scope="module", autouse=True)
+def set_selected_tor_active(duthosts, rand_one_dut_hostname):
```

Comment:
Because it is an autouse fixture, can we define this fixture in tests/common/dualtor/dual_tor_utils.py? Otherwise, it is executed for all topologies.

Reply:
Good suggestion! Will update. Thanks!